generators/cmd/cli/cmd/write.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"generators/pkg/writer"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var specFilename string
+var outputFilename string
+var writeCmd = &cobra.Command{
+	Use:   "write",
+	Short: "Write generated configuration to disk.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(specFilename) == 0 {
+			cmd.Help()
+			return nil
+		}
+
+		if len(outputFilename) == 0 {
+			outputFilename = "./gen-task.yaml"
+		}
+
+		file, err := os.Create(outputFilename)
+		if err != nil {
+			return err
+		}
+		defer file.Close()
+
+		err = writer.WriteToDisk(specFilename, file)
```

Comment:
It's generally a good practice to use the inline syntax where possible:

```
if err := ... ; err != nil {
  ...
}
```

It helps limit scope of variables where possible to avoid potential issues when reusing var names.

Reply:
Alternatively, you can just `return writer.WriteToDisk`